src/templates/community-champion-profile.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Breadcrumb from '~components/dev-hub/breadcrumb';
+import Layout from '~components/dev-hub/layout';
+import { fontSize, screenSize, size } from '~components/dev-hub/theme';
+
+const BREADCRUMB_BOTTOM_MARGIN = '40px';
+
+const ContentContainer = styled('div')`
+    padding: ${size.default} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        padding: ${size.mediumLarge} ${size.default};
+    }
+`;
+
+const StyledBreadcrumb = styled(Breadcrumb)`
+    line-height: 1;
+    margin-bottom: ${BREADCRUMB_BOTTOM_MARGIN};
+    > a {
+        font-size: ${fontSize.xsmall};
```

Comment:
Just curious is this different from the rest of the breadcrumbs on the platform?

Reply:
no, the breadcrumbs in hero banners also use this font size